src/ui/cmake/Dependencies.cmake

```diff
@@ -49,7 +49,7 @@ FetchContent_Declare(
 FetchContent_Declare(
         graph-prototype
         GIT_REPOSITORY https://github.com/fair-acc/graph-prototype.git
-        GIT_TAG 7d22e962c7f67fee95fd98c38bdb32ccde78d616
+        GIT_TAG fb8b080767ce47789a30c226f2a2ad51b8e2430a # main as of 2024-01-31
 )
```

Comment:
This whole block should actually be deleted. It's not doing anything since graph-prototype is already declared in the outer CMakeLists.txt.

Reply:
Are you sure? If I remove opencmw-cpp and graph-prototype here (both in the outer cmake/Dependencies.cmake), I get errors related to CMakeRC, and fmt not found etc....
I think that's because the native build builds src/ui separately using emscripten. (That's why we cannot have proper helper library targets on top-level)